src/main/resources/db/change-log.xml

```diff
@@ -33,6 +33,29 @@
             </column>
         </createTable>
     </changeSet>
+    <changeSet id="category" author="Borys">
+        <createTable tableName="category">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" />
+            </column>
+            <column name="parentId" type="BIGINT">
+                <constraints nullable="false" unique="false" />
```

Comment:
nullable = false. A może jednak nie i wtedy null sygnalizuje root? Czy może macie inny pomysł na wskazanie root'a?

Reply:
To był jeden z pomysłów jakie przyszły mi do głowy, choć nie wiedziałem, czy jest dobry :) 
Podoba mi się to rozwiązanie.

Reply:
Zmienione